### PR TITLE
[infra] Update docker image build command

### PR DESCRIPTION
**Pull request by hseok-oh**
- Introduce ubuntu 20.04 dockerfile
- Remove deprecated command parameter: "--extension"

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: infra/docker/focal/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (seanshpark) [inline infra/docker/focal/Dockerfile:1]: maybe 2020?

Comment (hseok-oh) [reply infra/docker/focal/Dockerfile]: I've updated. Thanks.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**